fratikcoiny/src/main/java/pl/fratik/fratikcoiny/libs/chinczyk/Chinczyk.java

```diff
@@ -600,8 +609,7 @@ private void timeout() {
             if (status == Status.IN_PROGRESS) {
                 Player player = players.get(turn);
                 player.setStatus(PlayerStatus.LEFT);
-                if (player.getControlHook() != null) player.getControlHook()
-                        .editOriginal(new MessageBuilder(t.get(player.getLanguage(), "chinczyk.left.timeout")).build()).complete();
+                if (player.getControlHook() != null) player.getControlHook().editOriginal(MessageEditData.fromContent(t.get(player.getLanguage(), "chinczyk.left.timeout"))).complete();
```

Comment:
Przy timeoucie przypadkiem nie zostaną przyciski? Na wszelki imo
```suggestion
                if (player.getControlHook() != null) player.getControlHook().editOriginal(MessageEditData.fromContent(t.get(player.getLanguage(), "chinczyk.left.timeout")).setReplace(true)).complete();
```